modules/wallet/seed_test.go

```diff
@@ -278,57 +278,71 @@ func TestSweepSeedFunds(t *testing.T) {
 	// wallet. This wallet should have a siafund balance.
 	//
 	// TODO: when proper seed loading is implemented, this will be unnecessary.
-	w, err := New(wt.cs, wt.tpool, wt.wallet.persistDir)
+	wt.wallet, err = New(wt.cs, wt.tpool, wt.wallet.persistDir)
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer w.Close()
 	// reset the changeID
-	resetChangeID(w)
-	err = w.Unlock(wt.walletMasterKey)
+	resetChangeID(wt.wallet)
+	err = wt.wallet.Unlock(wt.walletMasterKey)
 	if err != nil {
 		t.Fatal(err)
 	}
-	_, siafundBal, _ := w.ConfirmedBalance()
+	_, siafundBal, _ := wt.wallet.ConfirmedBalance()
 	if siafundBal.Cmp(types.NewCurrency64(2000)) != 0 {
 		t.Error("expecting a siafund balance of 2000 from the 1of1 key")
 	}
+	// need to reset the miner as well, since it depends on the wallet
+	wt.miner, err = miner.New(wt.cs, wt.tpool, wt.wallet, wt.wallet.persistDir)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// send funds to ourself a few times
+	for i := 0; i < 10; i++ {
+		uc, err := wt.wallet.NextAddress()
+		if err != nil {
+			t.Fatal(err)
+		}
+		_, err = wt.wallet.SendSiafunds(types.NewCurrency64(1), uc.UnlockHash())
+		if err != nil {
+			t.Fatal(err)
+		}
+		wt.addBlockNoPayout()
+	}
+	// send some funds to the void
+	_, err = wt.wallet.SendSiafunds(types.NewCurrency64(10), types.UnlockHash{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	wt.addBlockNoPayout()
```

Comment:
do you think it's worth trying to achieve some DRYness with helper functions, like we've done in the past, or just copy-paste everything?

Another incentive to combine tests is to avoid doing the same work twice, e.g. if a test takes 8 seconds to set up. But you can usually mitigate that via `t.Parallel`.